site/documentation/content/dev-guide/third_party_adapter.md

```diff
@@ -0,0 +1,263 @@
++++
+title = "Implementing Third Party Adapters"
+weight = 390
++++
+
+Eclipse Hono&trade; has the concept of protocol adapters (check the [Component Overview]({{< relref "architecture/component-view/index.md" >}}) for details),
+which provide protocol endpoints for devices.
+Hono already comes with a set of protocol adapters for some of the well known IoT protocols.
+However, in some cases devices do not communicate via standardized protocols but use a proprietary protocol.
+To provide the service user of a Hono service the possibility to also connect devices speaking a proprietary protocol
+Hono supports the concept of *Third Party Adapters*.
+
+A *Third Party Adapter* is a separately deployed micro service, which understands proprietary protocol messages and translates
+them to AMQP messages processable by Hono's AMQP protocol adapter.
+In that sense a *Third Party Adapter* behaves like a gateway for the devices speaking the proprietary protocol and forwards their
+messages to the AMQP Messaging Network. 
+
+The diagram below provides an overview of the *Third Party Adapter* setup.
+
+{{< figure src="../third_party_adapter.png" >}}  
+
+Devices send telemetry/event messages via their own proprietary protocols to the *Third Party Adapters* gateway,
+which send the messages via AMQP to the Hono AMQP adapter.
+
+A prerequisite for using *Third Party Adapters* is that devices are registered in Hono's Device Registry
+and are configured in such a way that the *Third Party Adapters* can act on their behalf i.e.
+with a device registration "via" property containing the gateway id of the *Third Party Adapters*.
+
```

Comment:
FMPOV this should go into a separate document under *concepts*. I can imagine something like *Connecting Devices* in which we describe the different options for how devices connect to Hono, e.g. directly or via a gateway. There we could then also introduce the notion of what you call here *Third Party Adapter* but which we actually call a *Protocol Gateway* as it is not a protocol adapter but instead is just a standard *gateway* which is potentially able to scale out horizontally.

Reply:
That sounds good. I took the naming from the issue, but find *Protocol Gateway* also better describing the module.